pkg/backends/calico/calico_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package calico
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestCalico(t *testing.T) {
```

Comment:
We have a couple other hooks we typically put here.

See here for an example: https://github.com/projectcalico/libcalico-go/blob/master/lib/ipam/ipam_suite_test.go#L27